dotcom-rendering/src/components/FollowButtons.tsx

```diff
@@ -0,0 +1,160 @@
+import { css } from '@emotion/react';
+import type { EmotionJSX } from '@emotion/react/types/jsx-namespace';
+import { remSpace, textSans } from '@guardian/source-foundations';
+import {
+	SvgCheckmark,
+	SvgNotificationsOff,
+	SvgNotificationsOn,
+	SvgPlus,
+} from '@guardian/source-react-components';
+import { decidePalette } from '../lib/decidePalette';
+
+type IconProps = {
+	isFollowing: boolean;
+	format: ArticleFormat;
+	iconIsFollowing: EmotionJSX.Element;
+	iconIsNotFollowing: EmotionJSX.Element;
+};
+
+const FollowIcon = ({
+	isFollowing,
+	format,
+	iconIsFollowing,
+	iconIsNotFollowing,
+}: IconProps): EmotionJSX.Element => (
+	<div
+		css={css`
+			background-color: ${isFollowing
+				? decidePalette(format).fill.shareIcon
+				: // TODO use decidePalette
+				  'white'};
+			height: 24px;
+			width: 24px;
+			margin: 0 0.3ch 0 0.5ch;
+			${isFollowing
+				? 'padding: 0.35ch 0 0 0.3ch;'
+				: 'padding: 0.25ch 0 0  0.2ch;'}
+			border-radius: 100%;
+			${!isFollowing && 'border: 1px solid;'}
+			border-color: ${!isFollowing &&
+			decidePalette(format).text.articleLink};
```

Comment:
what would the value of `border-color` be if `isFollowing` is `true`?

Reply:
It shouldn't exist. I will refactor lines 38-40 so that's clearer in the CSS